test/tasks-test.coffee

```diff
@@ -0,0 +1,87 @@
+SinonChai = require('sinon-chai')

```

Comment:
I think the file name speaks for themselves. Making it more granular on the folder structure level would just make it slightly more work to traverse.